src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -78,13 +72,19 @@ public void execute_storageThrowsIoException_throwsCommandException() {
         logic = new LogicManager(model, storage);
 
         // Execute add command
+<<<<<<< HEAD
+        String addCommand = AddCommand.COMMAND_WORD + NAME_DESC_TURKEY_SANDWICH + TIME_DESC_TURKEY_SANDWICH +
+        //EMAIL_DESC_TURKEY_SANDWICH;
+        Recipe expectedRecipe = new RecipeBuilder(TURKEY_SANDWICH).withGoals().build();
+=======
         String addCommand = AddCommand.COMMAND_WORD + NAME_DESC_AMY + TIME_DESC_AMY + STEP_DESC_AMY;
         Recipe expectedRecipe = new RecipeBuilder(AMY).withGoals().build();
+>>>>>>> master
```

Comment:
is this supposed to be like this (to resolve later) or did ya miss out the merge conflict here cuz it was commented out :o

Reply:
also I think I know why its failing!! For goal to be valid the criteria is alphanumerical w no spaces so once we change the criteria for it to match our 3 goals it should hopefully pass!! but yeah i can change it later just fyi hehe

Reply:
oops yes! i missed that merge conflict because it was commented out, my bad.
ahh, ive changed that already, the validation regex is now "^[ A-Za-z]+$+" which accepts alphabets and spaces

Reply:
the tests that are failing now are the add, edit, find commands which have not been refactored properly yet

Reply:
ahhh okay!